src/api/system.c

```diff
@@ -2,19 +2,25 @@
 #include <string.h>
 #include <stdbool.h>
 #include <ctype.h>
-#include <dirent.h>
-#include <unistd.h>
 #include <errno.h>
+#include <sys/types.h>
 #include <sys/stat.h>
 #include "api.h"
 #include "../rencache.h"
+#include "../utfconv.h"
```

Comment:
Should this moved inside the `#ifdef _WIN32` guard for clarity?

Reply:
Yes, it should